src/main/java/com/assembleia/votacao/service/PautaService.java

```diff
@@ -32,7 +33,7 @@ public Pauta deletaPauta(Long id){
             repository.deleteById(id);
             return pauta.get();
         }else {
-            throw new RuntimeException("Pauta não existe");
```

Comment:
pauta não existente se enquadra no not found e não badrequest, altera pra nós manin